packages/arb-provider-ethers/src/lib/value.ts

```diff
@@ -612,11 +609,6 @@ function _unmarshalValue(array: Uint8Array, offset: number): [Value, number] {
     const nextHash = ethers.utils.hexlify(head)
     return [new CodePointValue(pc, op, nextHash), offset]
   } else if (ty === ValueType.HashOnly) {
-    ;[head, offset] = extractBytes(array, offset, 8)
-    const size = ethers.utils.bigNumberify(head)
-    ;[head, offset] = extractBytes(array, offset, 32)
-    const hash = '0x' + head
-    // return [new HashOnlyValue(hash, size), tail];
     throw Error('Error unmarshaling: HashOnlyValue was not expected')
```

Comment:
sanity check here — is this the intent here indeed to always throw an error for this condition?

Reply:
Yep 😁